### PR TITLE
FOCUS #31: Invoicing Entity spec changes and supporting documents

### PR DESCRIPTION
**Pull request by ijurica**
Invoicing Entity spec changes and supporting documents

### CODE REVIEW

File: specification/dimensions/invoicingentity.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Provider
+
+An Invoicing Entity is an entity responsible for invoicing for the infrastructure and/or services consumed. It is commonly used for cost analysis and reporting scenarios.
+
+The InvoicingEntity column MUST be present in the billing data. This column MUST be of type String and MUST NOT contain null values.
```
Comment (mike-finopsorg) [inline specification/dimensions/invoicingentity.md:5]: should there be a space in InvoicingEntity?

File: specification/dimensions/invoicingentity.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+See [Appendix: Origination of the cost data](../appendix/origination_of_cost_data.md) section for context (Provider/Publisher/Invoicing Entity) in various purchasing scenarios.
```
Comment (mike-finopsorg) [inline specification/dimensions/invoicingentity.md:7]: We need to confirm how this link works in the spec PDF

File: specification/dimensions/invoicingentity.md (hunk 1 above)
Comment (ijurica) [reply specification/dimensions/invoicingentity.md:5]: discussed and chose to refer to the column ID, and not the display name in normative content, at least for now

File: specification/dimensions/invoicingentity.md (hunk 2 above)
Comment (udam-f2) [reply specification/dimensions/invoicingentity.md:7]: It won't. I added comment in the other PR for this. We need to use the anchor.

File: specification/dimensions/invoicingentity.md (hunk 1 above)
Comment (udam-f2) [reply specification/dimensions/invoicingentity.md:5]: So here where we're at with this (and definitely open to changing):

The first sentence is referring to the 'thing', thus using the column display name as a pro noun. I could be wrong in this. Michael Flanakin thought it should be lower case (e.g. "An invoice issuer is an entity responsible for". Alternatively we can use the Column ID. Which of the 3 options is the right one? Im not sure, but agree we need to just use one way. Can easily update other places once we decide.

The second sentence is normative content that uses the Column ID. It's attempting to refer to the column and some times the value inside the column (in the case of normalized columns where we say must be one of the following values). Again, open to an approach which we can then update in all others.

Any opinions (possibly from an expert) on this?

**Comment by ijurica** [comment]
Short overview of the currently applied changes and leftovers...

**APPLIED CHANGES:**

- Due to dimension renaming (Invoicing Entity -> Invoice Issuer), the MD files were renamed and every mention of the terms Invoicing Entity and InvoicingEntity were replaced.
- Current ordering in the dimensions.mdpp file - invoiceissuer.md placed above servicename.md
- Two mentions of the term "infrastructure and/or services" were replaced with the term "resources and/or services".
- In the spec MD file, link to the corresponding appendix section uses anchor (#origination-of-cost-data). Can be easily changed to (#originationofcostdata).
- The proposed change, related to the wording of the reference to the appendix section in the supporting content MD file (see conversation in https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/11 Provider PR), has been implemented.

**LEFTOVERS:**

- Column ID vs. Display Name within normative content - see unresolved conversation in https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/11 Provider PR
- Note regarding dimension's common usage scenarios - see unresolved conversation in https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/issues/11 Provider PR

File: specification/dimensions/invoicingentity.md (hunk 2 above)
Comment (ijurica) [reply specification/dimensions/invoicingentity.md]: changed to (#originationofcostdata)

**Review by udam-f2 — APPROVED** [review]
LGTM

File: specification/dimensions/invoicingentity.md (hunk 1 above)
Comment (udam-f2) [reply specification/dimensions/invoicingentity.md]: Will fix this more broadly in all docs given the decision from ChargeType PR

Comment (udam-f2) [reply specification/dimensions/invoicingentity.md]: Will fix this more broadly in all docs given the decision from ChargeType PR

**Pull request merged**